mirdata/medleydb_melody.py

```diff
@@ -156,12 +156,9 @@ def audio(self):
         return librosa.load(self.audio_path, sr=None, mono=True)
 
     def to_jams(self):
+        # jams does not support multipitch, so we skip melody3
         return jams_utils.jams_converter(
-            f0_data=[
-                (self.melody1, 'melody1'),
-                (self.melody2, 'melody2'),
-                (self.melody3, 'melody3'),
```

Comment:
i'm not sure how `self.melody3` is stored, but couldn't we write a `jams_converter` for it? it seems to me that JAMS does support multipitch.

<img width="372" alt="Screen Shot 2020-03-05 at 6 51 11 PM" src="https://user-images.githubusercontent.com/3943142/76038903-d673f980-5f18-11ea-8840-132d7b9dabf9.png">

Reply:
yep you're right. Writing the change now

Reply:
update: actually, it seems impossible, because JAMS is row-major (sources first, time frames later) whereas mir_eval is column-majoe (time frames first, sources later). This causes an inherent ambiguity in the conversion of the ragged array, because there is no deterministic one-to-one mapping between sources.

It would be good to document this limitation of `MultipitchData`